### PR TITLE
Fix deprecated setting `true` for `action_dispatch.show_exceptions`

### PR DESCRIPTION
**Pull request by jclusso**

### CODE REVIEW

**Comment by olivier-thatch** [comment]
@yuki24 Could this be merged in? Without this change, using the `with_exceptions_app` test helper with a Rails 7.1 app produces ugly warnings:
```
DEPRECATION WARNING: Setting action_dispatch.show_exceptions to true is deprecated. Set to :all instead. (called from block (3 levels) in <class:RambulanceTest> at test/initializers/rambulance_test.rb:51)
```

**Comment by yuki24** [comment]
Thanks @jclusso for your PR and @olivier-thatch for the heads up. I wanted to cut a new release sooner than later so I made a change by myself rather than leaving feedback here. Regardless, I really appreciate your time and effort. Thank you!

File: lib/rambulance/test_helper.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def with_exceptions_app(enabled: true)
       org_show_exceptions          = Rails.application.env_config['action_dispatch.show_exceptions']
 
       Rails.application.env_config['action_dispatch.show_detailed_exceptions'] = !enabled
-      Rails.application.env_config['action_dispatch.show_exceptions']          = enabled
+      Rails.application.env_config['action_dispatch.show_exceptions']          = enabled ? :all : org_show_exceptions
```
Comment (yuki24) [inline lib/rambulance/test_helper.rb:31]: This should not fall back to the `org_show_exceptions`, but rather it should be some sort of feature detection. You can see the actual fix here: https://github.com/yuki24/rambulance/pull/75/files#diff-2ab79f2a64326c3e9e254b6f1494dde94ef02ef753c9777a620fc845397aaa8aL31-R36

**Comment by yuki24** [comment]
This should be fixed with the newly released version 3.1.0.

**Comment by olivier-thatch** [comment]
Thank you! 🚀

**Pull request closed without merging**